doc/whats_new/v1.1.rst

```diff
@@ -525,6 +525,13 @@ Changelog
   parameters in `fit` instead of `__init__`.
   :pr:`21436` by :user:`Haidar Almubarak <Haidar13 >`.
 
+:mod:`sklearn.tree`
+.......................
+
+- |Fix| Fix a bug in the Poisson splitting criterion for
+  :class:`tree.DecisionTreeRegressor`.
+  :pr:`22191` by :user:`Christian Lorentzen <lorentzenchr>`.
+
```

Comment:
oops I realize that I merged too quickly: the `sklearn.svm` section has been split. Let me open a PR to fix this.